pkg/framework/aries/api/factory.go

```diff
@@ -8,9 +8,15 @@ package api
 
 import (
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/didresolver"
 )
 
 // TransportProviderFactory allows overriding of aries protocol providers
 type TransportProviderFactory interface {
 	CreateOutboundTransport() transport.OutboundTransport
 }
+
+// DIDResolverProviderFactory allows overriding of aries did resolver providers
+type DIDResolverProviderFactory interface {
```

Comment:
What is a "factory"? Do we now have Factory -> Provider -> DidResolver?

Reply:
Removed this dependency.